pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -815,28 +815,63 @@ cdef class _Timedelta(timedelta):
 
     cpdef timedelta to_pytimedelta(_Timedelta self):
         """
-        return an actual datetime.timedelta object
-        note: we lose nanosecond resolution if any
+        Converts a Timedelta object into a python datetime.timedelta object.
+
+        Timedelta objects are typecast to numpy datetime64[ns] dtype,
```

Comment:
`internally saved as` instead of `typecast to` would wound better to me

I'd use a stop mark to separate the second sentence instead of a comma, would make this easier to read.

`convert back` -> `convert`